enterprise/models.py

```diff
@@ -272,6 +272,37 @@ def user_email(self):
             return self.user.email
         return None
 
+    @property
+    def entitlements(self):
```

Comment:
@haikuginger yes, I think we should also have course specific entitlements in the response. Here is the sample response for entitlements after the modifications, let me know your thoughts on this
```
GET /enterprise/api/v1/enterprise-learner/1/entitlements/

{
    "global": [],
    course_specific: {
        "course-id-1": [5],
        "course-id-2": [2]
    }
}
```
However, I am a bit confused about using `EnterpriseCourseEnrollment` for consent as  `EnterpriseCourseEnrollment` model contains entries  that correspond to the consent for course enrollment, meaning the learner has already enrolled in the course, so, should the learner still be eligible for the entitlements ? (if learner somehow wants to enroll in the same course twice). I think it would make more sense if we had course specific consent rather than enrollment specific consent.
I think we should not be returning course specific entitlements using `EnterpriseCourseEnrollment` model as it does not entail course specific consent but an enrollment specific consent. let me know your thoughts on this.
@mattdrayer -- FYI